### ISSUE TITLE
[Discussion] Show banned hashtag on `/federation`

### ISSUE THREAD

**Issue by BentiGorlich** [MbinOrg/mbin#697]
Generally speaking, I am no fan of hiding potentially crucial information from a user. This is the same with federated/blocked instances. Knowing what type of content you can and can't engage with is about the most important factor when choosing an instance to create an account on and not exposing that crucial information to the user is fatigueing.

I for one joined an Mbin instance, precisely because `/federation` (also link `/instances` to `/federation` for parity with Lemmy pls, k thx bai) shows me what other instances are blocked. In that sense, I'd like to argue to go even a step further and show blocked tags under `/federation` as well.

_Originally posted by @UmBottesWillen in https://github.com/MbinOrg/mbin/issues/654#issuecomment-2045114812_

**Comment by BentiGorlich** [comment]
> Generally speaking, I am no fan of hiding potentially crucial information from a user. This is the same with federated/blocked instances. Knowing what type of content you can and can't engage with is about the most important factor when choosing an instance to create an account on and not exposing that crucial information to the user is fatigueing.

I disagree with that statement. Yes for a user it might be some important information, however it is also very important information for bad actors. There is a reason that you can hide the blocked instances on mastodon for anonymous users. It is just a good reason for bad actors / instances to dogpile on someone who decides to defederate for whatever reason from whatever instance. 
I don't know whether I would want to show all the banned hashtags to be publicly available. I don't want every banned user to be available on the federation page either... So it is a difficult line to walk on, that's for sure.

**Comment by UmBottesWillen** [comment]
If the hiding of blocked hashtags is opt-in, then I'm ok with that. That would disqualify an instance for me, but if the admin thinks that is necessary to protect themselves, so be it. I for one haven't heard about any admin getting dogpiled on for defederating from certain instances on Lemmy, which makes sense to me, given that you can see the blocked instances before creating an account and thus choose a different instance.    
 
If anything, I feel that having to create an account first to see blocked hashtags or even having to test out if any hashtags are blocked, will lead to more dogpiling, as users will have to create accounts first and invest time, only to find out that their effort was in vain. Some might choose to ask or even harass the admin to unblock a hashtag, rather than creating a new account on a different instance.

If the harassment of admins is the concern here, then I'd rather go the route of hiding any way to determine/contact the admin for anonymous users.
